support/src/indirect_intrusive_heap.h

```diff
@@ -261,7 +261,7 @@ namespace crimson {
     Iterator find(const I& ind_item) {
       for (HeapIndex i = 0; i < count; ++i) {
 	if (data[i] == ind_item) {
-	  return Iterator(*this, i);
+	  return Iterator(this, i);
```

Comment:
All of these constructor invocations for Iterator will go back to "*this".

Reply:
OK